db/migrate/20190404201812_create_field_export_rules.rb

```diff
@@ -0,0 +1,17 @@
+class CreateFieldExportRules < ActiveRecord::Migration[5.2]
```

Comment:
How would you feel about changing this migration name to `CreateFieldExportProfilesAndExportRules`?

Reply:
I don't feel like we need to change it. If you feel strongly about it, I can change it. I treat migration files as static once they are created.

Reply:
I'm good with keeping as is then.